investors/views.py

```diff
@@ -202,3 +211,64 @@ def perform_destroy(self, instance):
             extra={"saved_id": instance.pk, "by_user": self.request.user.pk},
         )
         super().perform_destroy(instance)
+
+class ViewedStartupPagination(pagination.PageNumberPagination):
+    """
+    Pagination class for recently viewed startups.
+    Default page size is 10, can be overridden via ?limit query parameter.
+    """
+    page_size = 10
+    page_size_query_param = "limit"
+    max_page_size = 50
+
+
+class ViewedStartupListView(generics.ListAPIView):
+    """
+    GET /api/v1/startups/viewed/
+    Retrieve a paginated list of recently viewed startups for the authenticated investor.
+    """
+    serializer_class = ViewedStartupSerializer
+    permission_classes = [IsAuthenticated, IsInvestor]
+    pagination_class = ViewedStartupPagination
+
+    def get_queryset(self):
+        return ViewedStartup.objects.filter(investor__user=self.request.user).order_by("-viewed_at")
```

Comment:
Filtering on `investor__user=self.request.user` is inefficient if there are multiple investors per user or if not indexed. Filter directly via the user's related investor object.
```
return ViewedStartup.objects.filter(investor=self.request.user.investor).order_by("-viewed_at")
```